src/services/API/Plio.js

```diff
@@ -15,6 +16,22 @@ export default {
   },
 
   getAllPlios() {
+    // returns the list of all plios
     return apiClient().get(process.env.VUE_APP_BACKEND_PLIOS_LIST);
   },
+
+  createPlio() {
+    // creates a new draft plio
+    var newPlioData = {
+      value: {
+        status: "draft",
+      },
+      bucket: "plio-test-data",
+    };
+    console.log(newPlioData);
+    return apiClient().post(
+      process.env.VUE_APP_BACKEND_CREATE_PLIO,
+      newPlioData
```

Comment:
Isn't it better to return a promise here, like it's being done above in the function `getPlioDetails`?